cli/src/build.ts

```diff
@@ -573,7 +591,7 @@ async function buildOnce(file: string, options: BuildOptions) {
             .forEach(fn => {
                 log(`  ${fn.name} (${prettySize(fn.size)})`)
                 fn.users.forEach(user =>
-                    debug(`    <-- ${resolver.posToString(user[0])}`)
+                    debug(`    <-- ${resolver.posToString(user[0])}`),
```

Comment:
is this , added by prettier? it kind of add noise to the PR

Reply:
yeah, must be prettier. Do you want to remove it?

Reply:
Yes, in general this kind of formatting changes really add noise to a PR and makes the review harder. If you have user settings overriding the repo .prettierrc file, you should disable them.